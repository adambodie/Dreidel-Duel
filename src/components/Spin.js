import React from "react"
import dreidel  from '../images/dreidel.gif'
import styles from "../css/main.module.scss";
const Spin =(props) => (
	<div className={styles.dreidel}>
		<img src={dreidel} alt="Dreidel"/>
		<h2>{props.spin}</h2>
	</div>
)

export default Spin;
