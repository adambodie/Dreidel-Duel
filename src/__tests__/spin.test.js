import React from "react"
import renderer from "react-test-renderer"
import Spin from "../components/Spin"
import {shallow} from "enzyme"

describe("Spin", () =>
	it("renders correctly", () => {
		const tree = renderer.create(<Spin />).toJSON()
		expect(tree).toMatchSnapshot()
	}),
	it('renders an `h2` text', () => {
		const wrapper = shallow(<Spin />);
		expect(wrapper.find('h2').length).toBe(1);
	})  
)
