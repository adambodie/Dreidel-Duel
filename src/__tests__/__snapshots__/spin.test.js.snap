// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spin renders correctly 1`] = `
<h2
  style={
    Object {
      "fontSize": 64,
      "width": 55,
    }
  }
/>
`;
